api/src/cli/utils/create-env/index.ts

```diff
@@ -16,18 +15,20 @@ const liquidEngine = new Liquid({
 	extname: '.liquid',
 });
 
-const defaults = {
-	security: {
-		KEY: uuid(),
-		SECRET: nanoid(32),
-	},
-};
-
 export default async function createEnv(
 	client: keyof typeof drivers,
 	credentials: Credentials,
 	directory: string
 ): Promise<void> {
+	const { nanoid } = await import('nanoid');
+
+	const defaults = {
+		security: {
+			KEY: uuid(),
+			SECRET: nanoid(32),
+		},
+	};
```

Comment:
Really tiny thing, but thoughts on combining this to `config` below (replacing `...defaults`)? Since they're in the same scope now. That way `config` will technically become the new "default". 🤔

Reply:
Makes sense 👍

Reply:
Yeet yeet skrrt skrrt